opendebates/templates/opendebates/snippets/add_question.html

```diff
@@ -23,7 +23,7 @@
             <select name="category">
               <option>--</option>
               {% for iter_category in SUBMISSION_CATEGORIES %}
-              <option {% if category and iter_category.id == category.id %}selected{% endif %}
+              <option {% if iter_category.id|slugify == form.data.category or iter_category.id == category.id %}selected{% endif %}
```

Comment:
(This seems terrible, but I didn't really want to waste a whole new variable just to compare an int and a string... is there a cleaner template-only way to do this?)